lib/enkf/res_config.cpp

```diff
@@ -239,6 +239,40 @@ res_config_type * res_config_alloc(const config_content_type * config_content) {
   return res_config;
 }
 
+res_config_type * res_config_alloc_full(char * config_dir,
```

Comment:
Could we not deduce this from the `user_config_file`?

Reply:
I think any parsing or business logic should be removed from the c code constructor.